pkg/controller/scheduler_common_test.go

```diff
@@ -1,5 +1,25 @@
 package controller
 
+import (
+	"fmt"
```

Comment:
nit: shoud we execute goimports?

Reply:
Indeed, I've added goimports to CI so PRs will fail from now on if imports are not properly formatted.

Thanks for the suggestion